v1/ao/internal/reporter/reporter_grpc.go

```diff
@@ -424,6 +456,9 @@ func (r *grpcReporter) eventSender() {
 	inProgress := false
 	var messages [][]byte
 
+	maxBatchTicker := time.NewTicker(r.eventMaxBatchInterval)
+	defer maxBatchTicker.Stop()
```

Comment:
Will the second `defer` statement override the first one?

Reply:
No, they will be executed in `Last In First Out` order.

Reply:
:+1: